cli/internal/graph/graph.go

```diff
@@ -12,17 +12,24 @@ import (
 )
 
 // WorkspaceInfos holds information about each workspace in the monorepo.
-type WorkspaceInfos map[interface{}]*fs.PackageJSON
+type WorkspaceInfos map[string]*fs.PackageJSON
```

Comment:
This was a decent big change. I think it was `interface{}` before because it was easier than using `string`, but it's always a string.

Reply:
Joys of Go (our version) not having generics so the generic set struct is actually just `type Set = map[inteface{}]interface{}` and this bubbles up :). I'm all for getting stuff back into concrete types sooner vs later 🎉